source/hardware/controllers/gemini_g4v.rst

```diff
@@ -0,0 +1,247 @@
+Gemini G4V
+=====================
+
+.. figure:: ../../_static/controllers/gemini_g4v_topview.jpg
+   :align: center
+   :width: 100%
+   :figwidth: 100%
+   :alt: Behringer CMD Micro
+   :figclass: pretty-figures
+
```

Comment:
```suggestion
```

Please remove, the other controller pages don't show photos of the controller either.